src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstanceDefinition.scala

```diff
@@ -213,64 +210,87 @@ object InstanceDefinition {
     )
 
   private def toInstancePropertyType(
-      instantPropTypeJson: Json,
+      propValue: Json,
       t: TypePropertyDefinition
   ): Either[DecodingFailure, InstancePropertyValue] =
     t.`type` match {
-      case PropertyType.DirectNodeRelationProperty(_) =>
-        Decoder[Seq[String]]
-          .decodeJson(instantPropTypeJson)
-          .map(InstancePropertyValue.StringList.apply)
-      case t if t.isList => toInstancePropertyTypeOfList(instantPropTypeJson, t)
-      case t => toInstancePropertyTypeOfNonList(instantPropTypeJson, t)
+      case _: PropertyType.DirectNodeRelationProperty =>
+        propValue
+          .as[Option[DirectRelationReference]]
+          .map(InstancePropertyValue.ViewDirectNodeRelation.apply)
+          .orElse {
+            // TODO: Remove this once this is fixed
+            // https://cognitedata.slack.com/archives/C031G8Y19HP/p1676895220201609
+            propValue.as[Option[List[String]]].flatMap {
```

Comment:
People are blocked by this.
Will remove this piece of code in a later PR, once they've fixed the bug